app/services/create_batch.rb

```diff
@@ -1,6 +1,8 @@
 class CreateBatch
   def call
-    unprocessed_appointment_summaries = AppointmentSummary.unprocessed.where(format_preference: :standard)
+    unprocessed_appointment_summaries = AppointmentSummary.unprocessed
+                                        .where(format_preference: :standard)
+                                        .where(country: [nil, '', 'United Kingdom'])
```

Comment:
Think its time to extract these into scopes. We're going to need it for manual processing of these in any case. Perhaps a separate PR though?